src/neonc.cpp

```diff
@@ -47,8 +47,13 @@ int main(int argc, char *argv[])
             if (i != std::string::npos) {
                 source_path = name.substr(0, i+1);
             }
-            std::cout << "Compiling " << name << "...\n";
             std::ifstream inf(name);
+            // TODO: Check for .neon file extension?  Maybe. 
+            if (not inf.good()) {
+                std::cout << "\"" << name << "\"" << " not found or is not a neon source file.\n";
```

Comment:
Also, surrounding the filename with double quotes here and not on the "Compiling ..." line is inconsistent.